src/ru/ashebalkin/skypro/lesson9/Library.java

```diff
@@ -0,0 +1,27 @@
+package ru.ashebalkin.skypro.lesson9;
+
+public class Library {
+    private final Book [] books;
+
+    public Library(int arrRange){
+        this.books = new Book [arrRange];
```

Comment:
```suggestion
        this.books = new Book[arrRange];
```
 рекомендую использовать ctrl+alt+L для автоформатирования кода